BlueprintUI/Tests/BlueprintViewTests.swift

```diff
@@ -112,9 +112,9 @@ private struct TestContainer: Element {
     var children: [Element]
     
     var content: ElementContent {
-        return ElementContent(layout: TestLayout()) { (builder) in
```

Comment:
Not a fan of this style. Loss of useful information here.

Reply:
I've taken to using the implicit $0 for methods under < 4 lines long, since its pretty clear what you're referring to